README.md

```diff
@@ -2,57 +2,61 @@
 
 `address_to_census_tract.py` prototypes address to Census tract conversion.  It uses a two-step process:
 
-1. Geocode a given single-line address to a (latitude, longitude) pair using a
-   remote (forward) geocoding service.
+1. Geocode a given single-line address to a (latitude, longitude) pair using a remote (forward) 
+   geocoding service.
```

Comment:
A lot of the changes to this README are re-wrapping lines from ~80 chars to a longer length. Doing so while making content changes makes the diff noisy and hard to read.

The actual content changes should also be part of the first commit on the branch.